src/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -67,41 +80,32 @@ function AdminNavbar({ targets, url_1 }: NavbarProps): JSX.Element {
     handleUpdateSpam();
   }, []);
 
-  useEffect(() => {
-    if (data && data.organizations[0].spamCount) {
-      setSpamCountData(
-        data.organizations[0].spamCount.filter(
-          (spam: any) => spam.isReaded === false
-        )
-      );
-    }
-  }, [data]);
-
-  const currentLanguageCode = Cookies.get('i18next') || 'en';
-
   const handleSpamNotification = (spamId: string) => {
     localStorage.setItem('spamId', spamId);
     window.location.assign(`/blockuser/id=${url_1}`);
   };
 
-  /* istanbul ignore next */
-  if (error) {
-    window.location.replace('/orglist');
-  }
```

Comment:
Don't remove error conditions, else tests would give warnings and errors. (Check above comment)